pkg/edgedictionaryitem/update.go

```diff
@@ -0,0 +1,50 @@
+package edgedictionaryitem
```

Comment:
As per my other comment, this file may need to be refactored to use the the pointer pattern.